atlasdb-ete-tests/src/test/java/com/palantir/atlasdb/ete/CassandraTimeLockTestSuite.java

```diff
@@ -44,5 +45,6 @@
             CassandraTimeLockTestSuite.class,
             "docker-compose.timelock.cassandra.yml",
             CLIENTS,
-            CassandraEnvironment.get());
+            CassandraEnvironment.get(),
+            true);
```

Comment:
can you do something similar to what you did with `ServiceCreator`?

Reply:
I added a new factory and removed devbreak to prevent future sadness